docs/platform/longview/longview-app-for-apache-classic/index.md

```diff
@@ -0,0 +1,322 @@
+    [FAIL] Reloading web server config: apache2 failed!
+  {{</ output >}}
+
+You will need to double-check your Apache installation, and then do a [manual configuration](#manual-configuration-all-distributions).
+
+### Unable to Access Local Server Status for Apache
+
+More specifically, the error will state `Unable to access local server status for Apache at <http://example.com/example?auto>: <error>:`. This error occurs when either:
+
+1.  Apache's `mod_status` setting is disabled or has been changed from the default location.
+
+2.  An Apache virtual host configuration is interfering with web requests to the `mod_status` location.
+
+ {{< note >}}
+This error occurs when Longview attempts to check the status page `location` listed in `/etc/linode/longview.d/Apache.conf`, or the default page at `127.0.0.1/server-status?auto`, but receives a non-200 HTTP response code. Basically, it means that the status page Longview is expecting is not being returned by the server.
+{{< /note >}}
+
+To fix this, follow these steps:
+
+1.  Make sure Apache is running:
+
+    Debian and Ubuntu:
+
+        service apache2 restart
+
+    CentOS and Fedora:
+
+        apachectl restart
+
+2.  Make sure `mod_status` is enabled. See the [Apache website](http://httpd.apache.org/docs/2.2/mod/mod_status.html#enable) for details. You can also check the output of the following command:
+
+        apachectl -M | grep status
+
+3.  Check the location for `mod_status`. The default location on Debian and Ubuntu systems is `http://127.0.0.1/server-status?auto` on localhost. In the Apache configuration file (typically `httpd.conf` or `status.conf`), this is designated with the lines:
+
+    {{< file "httpd.conf" apache >}}
+<Location /server-status>
+    SetHandler server-status
+
+{{< /file >}}
+
+    The `SetHandler server-status` line indicates that this is the location block for mod\_status. The location line itself sets the location.
+
+    ##### On cPanel/WHM
+
+    To direct Longview to the cPanel customized status page, edit the `location` line in `/etc/linode/longview.d/Apache.conf` to match the following:
+
+    {{< file "/etc/linode/longview.d/Apache.conf" apache >}}
+location http://localhost/whm-server-status?auto
+
+{{< /file >}}
+
+
+4.  Longview is designed to check the default location automatically. If you use the default location shown above, you should be done. Refresh the Longview Apache tab in the Linode Manager to verify that it's working now.
+5.  If you're not using the default location, you need to create a new file, `/etc/linode/longview.d/Apache.conf`, and set the `location` variable to match what you set in the Apache configuration file:
+
+    {{< file "/etc/linode/longview.d/Apache.conf" apache >}}
+location http://127.0.0.1/custom/location/path
+
+{{< /file >}}
+
+6.  Determine if an Apache virtual host configuration is interfering with requests to the mod_status location. Use a tool like `curl` or `wget` to request the server status location:
```

Comment:
mod_status should be in backticks

Reply:
this is classic manager guide...so disregarding